compiler/rustc_next_trait_solver/src/solve/normalizes_to/mod.rs

```diff
@@ -413,11 +414,12 @@ where
         panic!("trait aliases do not have associated types: {:?}", goal);
     }
 
-    fn consider_builtin_sized_candidate(
+    fn consider_builtin_sizedness_candidates(
         _ecx: &mut EvalCtxt<'_, D>,
         goal: Goal<I, Self>,
+        _sizedness: SizedTraitKind,
     ) -> Result<Candidate<I>, NoSolution> {
-        panic!("`Sized` does not have an associated type: {:?}", goal);
+        panic!("`Sized`/`MetaSized`/`PointeeSized` does not have an associated type: {:?}", goal);
```

Comment:
```suggestion
        panic!("Sized/MetaSized does not have an associated type: {:?}", goal);
```

Reply:
Removed `PointeeSized` here but kept the backticks because the other functions in this file all had them.